test/test_complex.py

```diff
@@ -33,5 +33,13 @@ def test_copy_real_imag_methods(self):
         self.assertEqual(complex_tensor.copy_real(), real)
         self.assertEqual(complex_tensor.copy_imag(), imag)
 
+    # issue: https://github.com/pytorch/pytorch/issues/36057
+    def test_bool_complex_add(self):
```

Comment:
it seems like this should just be another case or dtype entry in an existing test from test_type_promotion, this seems like a duplicate test